volkswagencarnet/vw_connection.py

```diff
@@ -541,6 +536,7 @@ async def post(self, url, vin="", tries=0, **data):
 
     # Construct URL from request, home region and variables
     def _make_url(self, ref, vin=""):
+        return ref
```

Comment:
After this line, the rest of the code is redundant?

Reply:
It would be hard to understand the intention in the future, so may be worth leaving a comment.

Reply:
Agreed! Will do that tomorrow